### PR TITLE
add the missing created_timestamp field in the provider api

### LINKED ISSUE #912: Missing creation date in sources
## User Story
As User, I want to see when the source was added so it will give me a good idea what I'm looking for.

Right now, the UI is using `customer.created_at` which is not the creation date of the source.

response

## Impacts
Ex. API, UI

## Role
User with permissions to view sources

## API Details

| API  | Detail|
 | --- | --- |
 | **Method** | GET |
 | **Path** | /api/v1/providers/ |
 | **Body** |  |
 | **Response** | {meta: {...},  links: {...},  data: [{ created_at: "...", ...  }, ...] } |

## UI Details
Using the created_at field in the provider data.

## Acceptance Criteria

- [ ] When getting providers (used in source settings) `created_at` will display the creation date of the provider/source

### PR DESCRIPTION
**Pull request by boaz0**
closes #912

### CODE REVIEW

**Comment by boaz0** [comment]
Thanks @blentz 
I will fix that

**Comment by adberglund** [comment]
@boaz0 If you look at my PR for cost model https://github.com/project-koku/koku/pull/910/files you can call it `created_timestamp` and that should be consistent.

File: koku/api/migrations/0011_auto_20190613_1554.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0010_costmodelmetricsmap'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='provider',
+            name='created_at',
```
Comment (adberglund) [inline koku/api/migrations/0011_auto_20190613_1554.py]:
```suggestion
            name='created_timestamp',
```

File: koku/api/provider/models.py [hunk 2]
```diff
@@ -84,6 +84,7 @@ class Meta:
     created_by = models.ForeignKey('User', null=True,
                                    on_delete=models.SET_NULL)
     setup_complete = models.BooleanField(default=False)
+    created_at = models.DateTimeField(auto_now_add=True, blank=True, null=True)
```
Comment (adberglund) [inline koku/api/provider/models.py]:
```suggestion
    created_timestamp = models.DateTimeField(auto_now_add=True, blank=True, null=True)
```

File: koku/api/provider/serializers.py [hunk 3]
```diff
@@ -76,6 +76,7 @@ class ProviderSerializer(serializers.ModelSerializer):
     name = serializers.CharField(max_length=256, required=True,
                                  allow_null=False, allow_blank=False)
     type = serializers.ChoiceField(choices=Provider.PROVIDER_CHOICES)
+    created_at = serializers.DateTimeField(read_only=True)
```
Comment (adberglund) [inline koku/api/provider/serializers.py]:
```suggestion
    created_timestamp = serializers.DateTimeField(read_only=True)
```

File: koku/api/provider/serializers.py [hunk 4]
```diff
@@ -86,7 +87,7 @@ class Meta:
 
         model = Provider
         fields = ('uuid', 'name', 'type', 'authentication', 'billing_source',
-                  'customer', 'created_by')
+                  'customer', 'created_by', 'created_at')
```
Comment (adberglund) [inline koku/api/provider/serializers.py]:
```suggestion
                  'customer', 'created_by', 'created_timestamp')
```

**Comment by boaz0** [comment]
sweet :yum: ! thanks @blentz @adberglund for your feedback and assistant.
Updated the PR to your requirements. Let me know if I'm missing anything.

**Comment by boaz0** [comment]
Thanks @blentz

**Pull request merged**